docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -32,45 +32,123 @@
 Inclusive Health Metrics: Race and Ethnicity
 ============================================
 
-On this page you will find ... 
+Overview
++++++++++++++++++++++
+
+On this page you will find ...
 
 .. todo::
-    
+
     Fill out this section to describe what this guide will include.
 
 If you have any questions or comments about the content on this page - please reach out! You can
-contact us directly `on GitHub <https://github.com/ihmeuw/vivarium_research/issues?q=is%3Aopen+is%3Aissue+label%3Abliss>`_ or anonymously 
-on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4QkFxJvno4WaEDz6h9rhJeyFlAlqyG7MAJg/viewform>`_.  
+contact us directly `on GitHub <https://github.com/ihmeuw/vivarium_research/issues?q=is%3Aopen+is%3Aissue+label%3Abliss>`_ or anonymously
+on our `feedback form <https://docs.google.com/forms/d/e/1FAIpQLSeCED9TFQsH-1u4QkFxJvno4WaEDz6h9rhJeyFlAlqyG7MAJg/viewform>`_.
 
 .. list-table:: **What this is vs. What this isn't**
    :header-rows: 1
 
    * - This is...
      - This isn't...
-   * -  
-     - 
+   * -
+     -
+
+Guiding principles
+------------------
 
-.. todo:: 
+.. todo::
 
     Fill out table with some guiding principles for this guide. Can refer to same table in the sex and gender guide.
 
+Introduction to race and ethnicity
+------------------------------------------
+
+Race and ethnicity are complex social constructs that have significant
+impacts on individuals' lives and health outcomes. While often used
+interchangeably, race and ethnicity are distinct concepts. Race is
+typically associated with physical characteristics such as skin color,
+facial features, and hair texture, while ethnicity encompasses cultural
+factors such as language, religion, and shared history.
+
+Historically, the concept of race has been used to justify
```

Comment:
Maybe it would be more accurate to say, "Historically, the concept of race was invented explicitly to justify discrimination, oppression, and social hierarchies."

Reply:
Agreed, and as I mentioned above, I think this should be part of how we define what race is.

Reply:
This paragraph feels more about the "what" to me, except for the last sentence, which is squarely in "why"!